README.rst

```diff
@@ -176,6 +176,13 @@ For example in your ``setup.cfg``:
     pytest_randomly.random_seeder =
         mypackage = mypackage.reseed
 
+or your ``pyproject.toml``:
+
+.. code-block:: toml
+
+    [tool.project.plugins."pytest_randomly.random_seeder"]
+    project = "project:randomly_reseed"
```

Comment:
🧵 [Quick search of GitHub](https://github.com/search?q=pytest_randomly+path%3A**%2Fpyproject.toml&type=code&ref=advsearch) yields this as the likely format but it didn't work in my testing. Will need to iterate here...

Reply:
Please put this in a separate PR.